CorsixTH/Lua/languages/english.lua

```diff
@@ -700,6 +701,7 @@ tooltip.hotkey_window = {
   button_recallPosKeys = "Assign hotkeys to save and recall camera positions",
   panel_debugKeys = "Assign hotkeys for debugging",
   button_back_02 = "Go back to the main hotkey window. Hotkeys changed in this window can be accepted there",
+  button_back = "Return to the options screen without saving changes",
```

Comment:
`hotkey_window.button_cancel` and `tooltip.hotkey_window.button_cancel` now become superfluous strings.
That's fine, but I would suggest checking the contents of the existing strings being removed, and see if they can be made to fit what already exists.

In my opinion here however, you can just delete them.

---

However, such a change does mean all other languages will lose this translation. I can rename the other translated languages' `hotkey_window.button_cancel` to `hotkey_window.button_back` so at least they have a translated string. We can then use our Discord to inform users of a string revision. **In short, you don't need to do this part of my comment yourself**